spec/services/mediator_spec.rb

```diff
@@ -25,7 +25,22 @@
       end
     end
 
-    context "with markdown file name" do
+    context "with a PDF file" do
+      let(:filename) { "spec/fixtures/files/gnu_manifesto.pdf" }
+      let(:file) { fixture_file_upload("gnu_manifesto.pdf") }
+
+      before do
+        allow(ConvertDocumentJob).to receive("perform_async")
+      end
+
+      it "converts document" do
+        subject.ingest(doc)
```

Comment:
I missed this last time: `use `described_class` when you're making a call to `Mediator`, and `subject` when you're making a call to an instance of `Meditator`.